### ISSUE TITLE
Dock link opens up separate icon not the link

### ISSUE THREAD

**Issue by ja601** [dmarmor/epichrome#231]
The mail and calendar Epichrome apps that I just created and dragged to the dock always open up a separate icon unlike in the earlier versions of Epi.

<img width="535" alt="Screenshot 2020-05-27 at 12 22 57" src="https://user-images.githubusercontent.com/23050789/83021417-4383c080-a02a-11ea-8d2b-389c32a371e0.png">

How can I fix that?

**Comment by talleux** [comment]
You need to create you Shortcut Icon in the Dock from the App under 
/Applications/Epichrome/EpichromeEngines.noindex/<Macos-USER>/<AppName>
not directly from /Applications/

Alternatively you just can drag the Icon from the right - when the app ist ope - to the left and remove the other one.

**Comment by dmarmor** [comment]
Thanks, @talleux, that's exactly what's going on. Another way to create a dock icon is to run an app, and then once it's running, right-click (or Control-click) on the dock icon and choose ```Options->Keep in Dock```.

**Comment by ja601** [comment]
> You need to create you Shortcut Icon in the Dock from the App under
> "/Applications/Epichrome/EpichromeEngines.noindex/[Macos-USER]/[AppName]"
> not directly from /Applications/
> 
> Alternatively you just can drag the Icon from the right - when the app ist ope - to the left and remove the other one.

Sweet tip. Worked like a charm. Merci so much talleux.

**Comment by talleux** [comment]
@ja601 your're welcome.
Note: if #232 happens to you then you need in this case to start the app from /Applications/[AppID] once, after that your Icon in /Applications/Epichrome/EpichromeEngines.noindex/[Macos-USER]/[AppID] should work again.
If then your custom icon is missing you need to set it again. I copy the icon from /Applications/[AppID] to /Applications/Epichrome/EpichromeEngines.noindex/[Macos-USER]/[AppID] using the "Info" menu from the finder for both app folder and copy&paste the icon

**Comment by talleux** [comment]
I have this behavior again on all apps since:
- i updated chrome to the latest version
- i updated Epichrome to 2.3.1 (from 2.3.0)
- i installed 2.3.2b2 BUT didn't update the apps to this version when prompted to

Unfortunately i did all 3 more or less on the same time so can't tell what causes this.

**Comment by dmarmor** [comment]
Hi @talleux, which behavior are you referring to here? Do you mean you're seeing duplicate dock icons?

**Comment by talleux** [comment]
@dmarmor yes. For me this issue is the sample as #226 and one of both could be closed. As well as #252 I think.

**Comment by dmarmor** [comment]
I'll keep this one open as I think the issue of the double icon in the dock is related but different enough from the generic icon issue. Thanks for the update!
